invenio_github/utils.py

```diff
@@ -91,7 +91,8 @@ def get_citation_metadata(gh, owner, repo_name, ref, release):
         return data
     except ValueError:
         raise CustomGitHubMetadataError(
-            file=current_app.config['GITHUB_CITATION_FILE'])
+            "We couldn't process the CITATION.cff file. Please "
```

Comment:
```suggestion
            file=current_app.config['GITHUB_CITATION_FILE'],
            "We couldn't process the CITATION.cff file. Please "
```

I think the `file` parameter is required. The second parameter is `message`, which can stay as is